client/trino-client/src/main/java/io/trino/client/ClientSession.java

```diff
@@ -368,9 +368,9 @@ public Builder authorizationUser(Optional<String> authorizationUser)
             return this;
         }
 
-        public Builder principal(Optional<String> principal)
+        public Builder user(Optional<String> user)
```

Comment:
nit: The current order bewteen fields and setters differs. I prefer using the same order.
```java
private Optional<String> user = Optional.empty();
private Optional<String> sessionUser = Optional.empty();
private Optional<String> authorizationUser = Optional.empty();
...
public Builder sessionUser(Optional<String> sessionUser)
public Builder authorizationUser(Optional<String> authorizationUser)
public Builder user(Optional<String> user)
```
Please feel free to ignore as pre-existing.

Reply:
I'll apply this change once the CI is done, thanks!